riscv-executor/src/lib.rs

```diff
@@ -724,12 +724,6 @@ mod builder {
 
             let main_degree = {
                 let range = namespace_degree_range(pil, "main");
-                assert!(
```

Comment:
Why can we remove those?

Reply:
for the continuations "dry run" it needs to run for the full execution (which will have a higher degree)